addons/base/tests/models.py

```diff
@@ -6,9 +6,11 @@
 from django.utils import timezone
 from framework.auth import Auth
 from framework.exceptions import HTTPError
-from nose.tools import (assert_equal, assert_false, assert_in, assert_is,
-                        assert_is_none, assert_not_in, assert_raises,
-                        assert_true)
+from nose.tools import (
+    assert_equal, assert_false, assert_in, assert_is,
+    assert_is_none, assert_not_in, assert_raises,
+    assert_true
```

Comment:
Trailing commas not enforced on imports?

Reply:
This is now supported. I upgraded the hook version in my latest commit.